### PR TITLE
Fix the dodgy footer position

### PR DESCRIPTION
**Pull request by morelazers**
Footer was borked and blocking a bunch of content - not sure if this breaks anything else

### CODE REVIEW

File: website/static/css/custom.css [hunk 1]
```diff
@@ -76,7 +76,7 @@ main {
 /* Try */
 
 html, body {
-  height: 100%;
```
Comment (jeluard) [inline website/static/css/custom.css]: Could be removed altogether? :)

**Comment by bhauman** [comment]
Glad you worked on this. I'll take a look at it.

File: website/static/css/custom.css (hunk 1 above)
Comment (bhauman) [reply website/static/css/custom.css]: looks like it

Comment (jeluard) [reply website/static/css/custom.css]: @morelazers

**Comment by jeluard** [comment]
@morelazers Thanks! Please squash your commits :)

**Comment by jeluard** [comment]
@morelazers Thanks! Just published the updated version

**Pull request merged**